proxy/hdrs/HTTP.cc

```diff
@@ -1264,9 +1264,14 @@ validate_hdr_content_length(HdrHeap *heap, HTTPHdrImpl *hh)
     int content_length_len         = 0;
     const char *content_length_val = content_length_field->value_get(&content_length_len);
 
-    // RFC 7230 section 3.3.2
+    // RFC 9110 section 8.6.
     // Content-Length = 1*DIGIT
     //
+    if (content_length_len == 0) {
```

Comment:
`content_length_len` is int, so while `value_get` doesn't actually ever return negative, people reading the call sires still have to keep that i mind.

Could we remove this version of `value_get` so that the unambiguous `string_view` version is the only version?

Reply:
Changed to `std::string_view`. PTAL.